[2022-05-24 12:47:18,727] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: book_db.save_books manual__2022-05-24T12:47:05.022533+00:00 [queued]>
[2022-05-24 12:47:18,741] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: book_db.save_books manual__2022-05-24T12:47:05.022533+00:00 [queued]>
[2022-05-24 12:47:18,742] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-24 12:47:18,742] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-05-24 12:47:18,743] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-24 12:47:18,766] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): save_books> on 2022-05-24 12:47:05.022533+00:00
[2022-05-24 12:47:18,773] {standard_task_runner.py:52} INFO - Started process 311 to run task
[2022-05-24 12:47:18,781] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'book_db', 'save_books', 'manual__2022-05-24T12:47:05.022533+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/books_db.py', '--cfg-path', '/tmp/tmpq0wm6t4j', '--error-file', '/tmp/tmpl1aip_5k']
[2022-05-24 12:47:18,786] {standard_task_runner.py:80} INFO - Job 9: Subtask save_books
[2022-05-24 12:47:18,885] {task_command.py:369} INFO - Running <TaskInstance: book_db.save_books manual__2022-05-24T12:47:05.022533+00:00 [running]> on host ac29b12a0730
[2022-05-24 12:47:19,030] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=book_db
AIRFLOW_CTX_TASK_ID=save_books
AIRFLOW_CTX_EXECUTION_DATE=2022-05-24T12:47:05.022533+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-24T12:47:05.022533+00:00
[2022-05-24 12:47:19,118] {python.py:173} INFO - Done. Returned value was: None
[2022-05-24 12:47:19,147] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=book_db, task_id=save_books, execution_date=20220524T124705, start_date=20220524T124718, end_date=20220524T124719
[2022-05-24 12:47:19,215] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-24 12:47:19,267] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
